_guides/analytics/deep_dive_analytics.md

```diff
@@ -0,0 +1,361 @@
+    </tr>
+    <tr>
+      <td data-th="Trigger Config"><code>timerSpec</code> (required when <code>on</code> is set to <code>timer</code>)</td>
+      <td data-th="Description">Controls when the <code>timer</code> event is fired. The timer will trigger immediately and then at a specified interval thereafter. This configuration is used on conjunction with the <code>timer</code> trigger.</td>
+    </tr>
+  </tbody>
+</table>
+
+**Important:** Triggers from a configuration with lower precedence are overridden
+by triggers with the same names from a configuration with higher precedence
+(see [Variable substitution ordering](/docs/guides/analytics/deep_dive_analytics.html#variable-substitution-ordering)).
+
+### How data gets sent: transport attribute
+
+The `transport` attribute specifies how to send a request.
+The following three methods are enabled by default:
+
+<table>
+  <thead>
+    <tr>
+      <th data-th="Transport Method" width="30%">Transport Method</th>
+      <th data-th="Description" width="70%">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td data-th="Transport Method"><code>beacon</code></td>
+      <td data-th="Description">Indicates <a href="https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon">navigator.sendBeacon</a> can be used to transmit the request. This will send a <code>POST</code> request, with credentials, and an empty body.</td>
+    </tr>
+    <tr>
+      <td data-th="Transport Method"><code>xhrpost</code></td>
+      <td data-th="Description">Indicates <code>XMLHttpRequest</code> can be used to transmit the request. This will send a <code>POST</code> request, with credentials, and an empty body.</td>
+    </tr>
+    <tr>
+      <td data-th="Transport Method"><code>image</code></td>
+      <td data-th="Description">Indicates the request can be sent by generating an <code>Image</code> tag. This will send a <code>GET</code> request.</td>
+    </tr>
+  </tbody>
+</table>
+
+Only one transport method gets used,
+and it's the one with the highest precedence
+that's enabled, permitted, and available.
+The precedence is `beacon` > `xhrpost` > `image`.
+If the client's user agent does not support a method,
+the next highest precedence method enabled gets used. 
+
+Include the `transport` attribute in your configuration
+only if you want to limit the transport options,
+otherwise, you may stop requests.
+
+In the example below,
+`beacon` and `xhrpost` are set to false,
+so they will not be used even though they have higher precedence than `image`.
+If the client's user agent supports the `image` method,
+then it will be used; otherwise, no request gets sent.
+
+{% highlight html linenos %}
+'transport': {
+  'beacon': false,
+  'xhrpost': false,
+  'image': true
+}
+{% endhighlight %}
+
+## Variable substitution ordering
+
+AMP populates variables with values in an order of precendence:
+Remote configurations > Triggers > Top-level > Platform.
```

Comment:
I'd use something other than "Top-level", or at least clarify it a bit.

- Remote configurations (via `config`)
- `vars` nested inside of a trigger within `triggers`
- `vars` at the top-level nested within `amp-analytics`
- Platform-provided values